### DISCUSSION TITLE
Better error messages...

### DISCUSSION THREAD

**Discussion by thebjorn** [sveltejs/kit#11865] [Ideas]
I just got this error:

    SvelteKitError: POST method not allowed. No actions exist for this page

I was trying to post from `/my/special/page` to `/logout?next=/goodbye`, so it's not exactly clear what "_this_ page" means in this context, especially if you're not experienced (e.g. does it mean a `+..page..` file, a path/url, or a route..?).

What can be improved:

1. give me all the information the error handler has - or can gather (it's already an error so there shouldn't be any concerns about using cycles to gather input).
2. give me an error number (`SvelteKitError[E042]: ...`) that I can look up to find more information (or even make the error number a link to the docs?)

**Comment by teemingc** [comment]
I think this is a great idea. I've created an issue for this here https://github.com/sveltejs/kit/issues/11876

**Comment by mig-hub** [comment]
I think it is a great idea as well. I am currently fighting the same issue and struggling to investigate because it does not tell me what it tries and what file/action could be missing.

**Reply by thebjorn** [reply]
You might want to comment on #11876 instead...
